cudamapper/src/overlapper.cpp

```diff
@@ -47,9 +48,22 @@ bool overlaps_mergable(const claraparabricks::genomeworks::cudamapper::Overlap o
         target_gap = (o2.target_start_position_in_read_ - o1.target_end_position_in_read_);
     }
 
-    auto gap_ratio    = static_cast<float>(std::min(query_gap, target_gap)) / static_cast<float>(std::max(query_gap, target_gap));
-    bool gap_ratio_ok = (gap_ratio > 0.8) || ((query_gap < 500) && (target_gap < 500)); //TODO make these user-configurable?
-    return gap_ratio_ok;
+    std::uint32_t o1_query_length  = o1.query_end_position_in_read_ - o1.query_start_position_in_read_;
```

Comment:
Can you add some documentation explaining the new algorithm and the logic behind it.

Reply:
Addressed in 4b30fe7.